### PR TITLE
feat: add fallback login page

### PR DESCRIPTION
**Pull request by netroms**
## Summary
Adds a simple fallback login page in pure HTML that uses the default Spring login endpoint.

## Manually test
1. Go to ./login.html
2. Log in as usual with your username/password. (this does not support 2FA)
3. Observe you get logged in.

### JIRA:
[DHIS2-17682](https://dhis2.atlassian.net/browse/DHIS2-17682)

[DHIS2-17682]: https://dhis2.atlassian.net/browse/DHIS2-17682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML>
+<html class="loginPage" dir="ltr">
+<head>
+  <title>DHIS 2 Demo</title>
```
Comment (david-mackessy) [inline dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html:4]: just a small thing, I've never seen a space between `DHIS` and `2`. Not sure if that is correct. Should it be `DHIS2`?

File: dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  <meta name="description" content="DHIS 2">
```
Comment (david-mackessy) [inline dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html:5]: same here

File: dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  <meta name="keywords" content="DHIS 2">
```
Comment (david-mackessy) [inline dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html:6]: same here

File: dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    });
+  </script>
+  <script type="text/javascript" src="./dhis-web-commons/javascripts/login.js"></script>
+  <link type="text/css" rel="stylesheet" href="./dhis-web-commons/css/widgets.css">
+  <link type="text/css" rel="stylesheet" href="./dhis-web-commons/css/login.css">
+</head>
+
+<body class="loginPage">
+<h1 style="display:none">DHIS 2</h1>
+<div style="display:none">DHIS 2</div>
```
Comment (david-mackessy) [inline dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html:32]: these 2 also and more below

**Review by vietnguyen — APPROVED** [review]

**Review by david-mackessy — APPROVED** [review]

**Pull request merged**